frontend/components/ifixit-image/ifixit-image.tsx

```diff
@@ -0,0 +1,35 @@
+import Image, { ImageProps, ImageLoader } from 'next/image';
+
+export function IfixitImage(props: ImageProps) {
+   let unoptimized = props.unoptimized;
+
+   if (typeof props.src === 'string') {
+      if (isGuideImage(props.src)) {
+         unoptimized = true;
+      } else if (isCartImage(props.src)) {
+         unoptimized = true;
+      } else if (isStrapiImage(props.src)) {
+         unoptimized = true;
+      }
+   }
```

Comment:
Might consider pulling the conditional below out into its own function, which allows setting `unoptimized` once:
```suggestion
   const unoptimized = props.unoptimized || !shouldOptimize(props.src);
```

Example `shouldOptimize`:
```ts
function shouldOptimize(src: any) {
   if (typeof src === 'string') {
      if (isGuideImage(src)) {
         return false;
      } else if (isCartImage(src)) {
         return false;
      } else if (isStrapiImage(src)) {
         return false;
      }
   }
   return true;
}
```

Reply:
In upcoming pulls we'll want to change both the loader *and* `unoptimized`. When that comes, we'll probably user a pattern like `const [unoptimized, loader] = getLoader(props)`